### PR TITLE
[DX-1437] standardise --> standardize, specialise --> specialize, utilise --> utilize, wilful --> willful

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/transform-traffic/request-method-transform.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Tyk's Request Method Transform middleware allows you to modify the HTTP method o
 
 In cases where an upstream API requires different methods (e.g. `PUT` or `DELETE`) for different functionality but you want to wrap this in a single client-facing API, you can provide a simple interface offering a single method (e.g. `POST`) and then use the method transform middleware to map requests to correct upstream method.
 
-#### Enforcing API governance and standardisation
+#### Enforcing API governance and standardization
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/transform-traffic/request-method-transform.md:16]: change is safe, no refs affected

File: tyk-docs/content/planning-for-production/monitoring/tyk-components.md [hunk 2]
```diff
@@ -16,13 +16,13 @@ A common question that gets asked is how to monitor the Tyk components.
 
 The Gateway & Redis are the only components that will have a high on-demand performance requirement, which needs to scale with your API traffic.
 
-### Tyk Gateway CPU Utilisation
+### Tyk Gateway CPU Utilization
```
Comment (dcs3spp) [inline tyk-docs/content/planning-for-production/monitoring/tyk-components.md:19]: change is safe, no refs to anchor affected

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ API Templates are exclusive to [Tyk OAS APIs]({{< ref "getting-started/key-conce
 #### Gateway agnostic API design
 When working with OpenAPI described upstream service APIs, your service developers do not need to learn about Tyk. You can create and maintain a suitable suite of templates that contain the Tyk-specific configuration (`x-tyk-api-gateway`) that you require for your externally published API portfolio. Creating an API on Tyk is as simple as importing the OpenAPI document and selecting the correct template. Tyk will combine the OpenAPI description with the template to produce a valid Tyk OAS API.
 
-#### Standardising API configuration
+#### Standardizing API configuration
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md:22]: change is safe, no refs to anchor affected

**Review by letzya — APPROVED** [review]

**Pull request merged**